source/pastemyst/db/db.d

```diff
@@ -80,6 +80,19 @@ public void insert(T)(T item)
     collection.insert(data);
 }
 
+/++
+ + Get the amount of Paste objects in the DB
+ +/
+public ulong getPastiesAmount(T)()
```

Comment:
This function actually can check the amount for any collection (since it's a template), so can just be renamed to `getCollectionCount`